texastribune/donations
# Forget about Auth0 usernames

#### What's this PR do?
Removes all Auth0 username references from the portal.

#### Why are we doing this? How does it help us?
Preparation for when we actually ditch that field on the Auth0 signup form. This PR can be merged before that because we decided it's good to start not showing people their Auth0 username in the portal's "contact info" section ASAP.

#### How should this be manually tested?
+ `make`
+ `make restart`
+ View `/account` while logged in as both a staff and non-staff test user. Confirm you see your email in both cases under "contact info." For staff, your first and last name should, without question, appear. For non-staff, they might or might not (depending on when you created your account). But you should _not_ see username.
+ When logged in as staff, try viewing-as someone and confirm it works

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
NA.

#### What are the relevant tickets?
Sort of this:
https://3.basecamp.com/3098728/buckets/10357222/todos/1899525968

#### Have you done the following, if applicable:
